src/Control/Email/Email.php

```diff
@@ -933,23 +967,29 @@ public function findPlainPart()
     }
 
     /**
+     * @deprecated 4.12.0 Will be replaced with Symfony\Component\Mime\Email::getTextBody()
```

Comment:
```suggestion
     * @deprecated 4.12.0 Use getTextBody() instead
```
`getTextBody()` sounds like it returns the actual body, rather than a boolean. If that's the case, we should probably keep this method and return `(bool) $this->getTextBody()` or similar.

Reply:
Actually it's probably more accurate for all the *PlainPart*() methods to say these are being removed without equivalent functionality

They are sort of replaced with the text() / getTextBody() methods that are in symfony, because the whole MIME Part thing has been abstracted away for us.

Reply:
Fair enough, please make that change then.

Reply:
Updated